code_generation/function.h

```diff
@@ -230,6 +230,16 @@ class KODE_EXPORT Function
      */
     QString docs() const;
 
+    /**
+     * Sets whether the function is marked with explicit specifier.
+     */
+    void setExplicit(bool isExplicit);
```

Comment:
Yes, it makes sense to add `= true` here, given that the default is obviously false. `setExplicit()` will read well.